examples/is31fl3731_wave_example.py

```diff
@@ -4,18 +4,43 @@
 
 i2c = busio.I2C(board.SCL, board.SDA)
 
-sweep = [1, 2, 3, 4, 6, 8, 10, 15, 20, 30, 40, 60, 60, 40, 30, 20, 15, 10, 8, 6, 4, 3, 2, 1]
+sweep = [
```

Comment:
I am documenting here that I question this change and support disabling Black formatting around this code block. I will however merge the PR. If in the future, those working on this wish to disable the formatter for this, please proceed.